modules/aws/dra-analytics/variables.tf

```diff
@@ -83,7 +83,7 @@ variable "archiver_user" {
 variable "dra_version" {
   description = "The DRA version to install"
```

Comment:
I suggest adding to the description - Supported versions are 4.11.0.10.0.7 and up. Similar to Sonar.